### PR TITLE
perf: vnc proxy port to 15900

### PR DESCRIPTION
**Pull request by fit2bot**
perf: vnc proxy port to 15900

### CODE REVIEW

File: apps/terminal/migrations/0005_endpoint_vnc_port.py [hunk 1]
```diff
@@ -16,6 +16,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='endpoint',
             name='vnc_port',
-            field=common.db.fields.PortField(default=5900, validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(65535)], verbose_name='VNC port'),
+            field=common.db.fields.PortField(default=15900, validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(65535)], verbose_name='VNC port'),
         ),
     ]
```
Comment (fit2cloudrd) [inline apps/terminal/migrations/0005_endpoint_vnc_port.py:21]: There is no need for any changes or adjustments to the current code. The provided migration file should run smoothly as it already adheres to all expected requirements, including adding a new column with the specified value range validation for the "vnc_port" field in the "endpoint" model table.

The implementation appears accurate based on its structure and usage, thus it does not exhibit abnormalities or require optimization.

File: apps/terminal/models/component/endpoint.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ class Endpoint(JMSBaseModel):
     postgresql_port = PortField(default=54320, verbose_name=_('PostgreSQL port'))
     redis_port = PortField(default=63790, verbose_name=_('Redis port'))
     sqlserver_port = PortField(default=14330, verbose_name=_('SQLServer port'))
-    vnc_port = PortField(default=5900, verbose_name=_('VNC port'))
+    vnc_port = PortField(default=15900, verbose_name=_('VNC port'))
 
     comment = models.TextField(default='', blank=True, verbose_name=_('Comment'))
     is_active = models.BooleanField(default=True, verbose_name=_('Active'))
```
Comment (fit2cloudrd) [inline apps/terminal/models/component/endpoint.py:28]: The code does not show any inconsistencies, issues, or specific optimizations that need to be suggested. It appears it's just setting up some fields for communication between JMSModel and PostgreSQL, Redis, SQL Server, VNC, etc. The values (such as 'postgresql', 'redis', 'sqlserver') have been set default values but nothing else indicates major issues like bugs, missing features, or improper logic within the structure of models. So no substantial changes are recommended currently.

However, remember, this check is based on knowledge cut-off information which may differ from current best practices, hence you might want to consider having discussions with more recent versions of the documentation or contacting official sources related to Django/Jobs/Messaging framework support to get accurate suggestions. 

In summary, there aren't immediate suggestions required right now; however, checking for modern coding standards (like using proper doc strings) should always be considered when writing clean, maintainable code.

**Review by ibuler — APPROVED** [review]
